### ISSUE TITLE
ihover in Jimdo?

### ISSUE THREAD

**Issue by atzezappa** [gudh/ihover#16]
Hi there,

I tried to implement the ihover code in a Jimdo web-page and it doesn´t work. Iwould be very thankful fo any advice why it doesn´t... here ist the address of the page:

http://ihovertest.jimdo.com/ihovertest/

So glad I found you,
Regards from Berlin, az

**Comment by cytec** [comment]
well... seems like its working to me

**Comment by atzezappa** [comment]
Yes thanks! I found the mistake in the CSS. It was  ...effectx a:hover instead of …effectx:hover which made it not work.

LG, az

 http://redesign-berlin.jimdo.com/ stempel-exp

Von: cytec [mailto:notifications@github.com] 
Gesendet: Dienstag, 9. Februar 2016 17:41
An: gudh/ihover
Cc: atzezappa
Betreff: Re: [ihover] ihover in Jimdo? (#16)

well... seems like its working to me
